scripts/azure-ci-test.sh

```diff
@@ -0,0 +1,90 @@
+#!/usr/bin/env bash
+
+set -o errexit
+set -o nounset
+set -o pipefail
+
+export GROUP_NAME="${GROUP_NAME:-ratify-e2e-$(openssl rand -hex 2)}"
+export ACR_NAME="${ACR_NAME:-ratifyacr$(openssl rand -hex 2)}"
+export AKS_NAME="${AKS_NAME:-ratify-aks-$(openssl rand -hex 2)}"
```

Comment:
Generate the random prefix once and reuse across all the variables so there is consistency